scss/blocks/figma-component-props-table.scss

```diff
@@ -0,0 +1,120 @@
+:root {
+    --figma-component-props-table-background-color: rgba(244, 245, 247, 0.95);
+    --figma-component-props-table-badge-border-color: #ECEEF3;
+}
+
+table.props-table {
+    width: 100%;
+}
+
+table.props-table thead {
+    background-color: var(--figma-component-props-table-background-color);
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 16px;
+    color: #606981;
+}
+
+table.props-table th {
+    border-top: none;
+    border-bottom: 1px solid #ECEEF3;
+}
+
+table.props-table th:first-child{
+    border-radius: 8px 0 0 0;
+}
+
+table.props-table th:last-child{
+    border-radius: 0 8px 0 0;
+}
+
+table.props-table td {
+    border-top: none;
+    border-bottom: 1px solid #ECEEF3;
+    padding: 12px;
+}
+
+table.props-table .props-table-value {
```

Comment:
shouldn't be there some different font family for this? some monospace based on desing

Reply:
You're right, forgot about this

Reply:
[09fc654](https://github.com/Supernova-Studio/exporter-documentation/pull/166/commits/09fc6540a3d9e3d740386835f10d704eded76576)